types/html-docx-js/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for html-docx-js 0.3
+// Project: https://github.com/evidenceprime/html-docx-js#readme
+// Definitions by: Jacob Fischer <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/html-docx-js/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me

Reply:
Woops. Thanks!

Reply:
Fixed in b8c391f1fd31291390c0aad4e65ec1afa0c79149